### PR TITLE
81 delete postpost idlike

### PR DESCRIPTION
**Pull request by DLiSWE**

### CODE REVIEW

File: src/services/post.service.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ class PostService {
   public async likePost({ id, userId }: LikePostDto): Promise<void> {
     await pgDataSource.createQueryBuilder().relation(PostEntity, 'likes').of(id).add(userId);
   }
+
+  public async deletePostLike(id: number, userId: number): Promise<void> {
```
Comment (ZohaibManzoor00) [inline src/services/post.service.ts:53]: same here, can we implement a DeletePostLikeDto?

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

**Review by ZohaibManzoor00 — APPROVED** [review]
This pr includes both deleting a like on a post and liking a comment. 
They look good - but avoid having 2 unrelated tickets in one pr

**Review by ZohaibManzoor00 — DISMISSED** [review]
Looks good

File: src/routes/post.route.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ class PostRoute implements Routes {
     this.router.delete(`${this.path}/:post_id`, AuthMiddleware, this.postController.deletePost);
     this.router.get(`${this.path}/:post_id/likes`, AuthMiddleware, this.postController.getLikesFromPost);
     this.router.post(`${this.path}/:post_id/likes`, AuthMiddleware, this.postController.likePost);
+    this.router.delete(`${this.path}/:post_id/:like_id`, AuthMiddleware, this.postController.deletePostLike);
```
Comment (raylu511) [inline src/routes/post.route.ts:23]: this doesn't follow rest convention. I think if you're just going to delete the like using userId and postId, then we can remove like_id

Comment (DLiSWE) [reply src/routes/post.route.ts]: wouldn't line 20 and 23 use the same path and method then?

Comment (DLiSWE) [reply src/routes/post.route.ts]: maybe just /likes?

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

**Review by raylu511 — APPROVED** [review]

**Pull request merged**